src/actions/action-types.js

```diff
@@ -0,0 +1,2 @@
+export const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
According to the project instructions on pathwright. you're supposed to use `src/actions/index.js`

Kindly correct the path, this looks nice but to be more professional about this, it good to abide by the React project structure.
![image](https://user-images.githubusercontent.com/31897434/106088598-91efde80-6126-11eb-9e8e-d2f50d633dfe.png)